### PR TITLE
Add backup and restore jobs

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

**Comment by jwilander** [comment]
@cpanato good for review right?

**Comment by cpanato** [comment]
yes @jwilander

File: mysql-backup/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Mattermost MySql Kubernetes Backup
```
Comment (jwilander) [inline mysql-backup/README.md]: Can we link to this from the main README?

**Comment by cpanato** [comment]
@jwilander PTAL

**Comment by cpanato** [comment]
@bananaspliff no, this is just the backup for the data in the mysql. for Minio you will need to snapshot the volume attached to minio and then this depends on which cloudprovider you are using

**Comment by bananaspliff** [comment]
@cpanato no, I would like to push mysql backup in minio instead of s3 (set empty region like in mattermost not working)

**Comment by cpanato** [comment]
@bananaspliff ah 😄 
I did not test, but maybe you need to change the `DB_DUMP_TARGET` to point to the minio address

**Comment by bananaspliff** [comment]
@cpanato  you can't, it's just for set the bucket

**Comment by cpanato** [comment]
I mean edit the manifest and adjust that for your needs

**Pull request merged**